Dockerfile

```diff
@@ -1,4 +1,5 @@
 ARG VERSION=latest
 FROM sonarqube:${VERSION}
 
-COPY target/sonar-clover-plugin.jar /opt/sonarqube/extensions/plugins/
\ No newline at end of file
+COPY target/sonar-clover-plugin.jar /opt/sonarqube/extensions/plugins/
+ADD https://github.com/Inform-Software/sonar-groovy/releases/download/1.8/sonar-groovy-plugin-1.8.jar /opt/sonarqube/extensions/plugins/
```

Comment:
Not sure that we can use an external lib. maybe should it be included in our own jar 🤔

Reply:
I'm not sure why I added the sonar-groovy-plugin, but I found something I believed was caused by the missing registration of the grvy language code.
I've tried to recreate the problem, but I can't, so it's not needed after all.
I'll remove it and amend the PR.